src/cbmc_starter_kit/template-for-ci-workflow/proof_ci.yaml

```diff
@@ -0,0 +1,246 @@
+            && cd kissat \
+            && git checkout $KISSAT_TAG_NAME \
+            && ./configure \
+            && cd build \
+            && make -j;
+          echo "$(pwd)" >> $GITHUB_PATH
+          echo "version=$KISSAT_TAG_NAME" >> $GITHUB_OUTPUT
+      - name: Install ${{ env.CADICAL_TAG }} cadical
+        id: install_cadical_version
+        if: ${{ env.CADICAL_TAG != '' }}
+        shell: bash
+        run: |
+          if ${{ env.CADICAL_TAG == 'latest' }}
+          then
+            CADICAL_REL="https://api.github.com/repos/arminbiere/cadical/releases/latest"
+            CADICAL_TAG_NAME=$(curl -s $CADICAL_REL | jq -r '.tag_name')
+          else
+            CADICAL_TAG_NAME=${{ env.CADICAL_TAG }}
+          fi
+          echo "Installing cadical $CADICAL_TAG_NAME"
+          git clone https://github.com/arminbiere/cadical.git \
+            && cd cadical \
+            && git checkout $CADICAL_TAG_NAME \
+            && ./configure \
+            && cd build \
+            && make -j;
+          echo "$(pwd)" >> $GITHUB_PATH
+          echo "version=$CADICAL_TAG_NAME" >> $GITHUB_OUTPUT
+      - name: List versions of installed tools
+        id: list_cbmc_proof_tool_versions
+        shell: bash
+        run: |
+          if [ -n "${{ steps.install_latest_cbmc.outputs.version }}" ]
+          then
+            echo "CBMC ${{ steps.install_latest_cbmc.outputs.version }}"
+          else
+            echo "CBMC ${{ steps.install_cbmc_version.outputs.version }}"
+          fi
+          if [ -n "${{ steps.install_latest_cbmc_viewer.outputs.version }}" ]
+          then
+            echo "CBMC viewer ${{ steps.install_latest_cbmc_viewer.outputs.version }}"
+          else
+            echo "CBMC viewer ${{ steps.install_cbmc_viewer_version.outputs.version }}"
+          fi
+          if [ -n "${{ steps.install_latest_litani.outputs.version }}" ]
+          then
+            echo "Litani ${{ steps.install_latest_litani.outputs.version }}"
+          else
+            echo "Litani ${{ steps.install_litani_version.outputs.version }}"
+          fi
+          if [ -n "${{ steps.install_kissat_version.outputs.version }}" ]
+          then
+            echo "kissat ${{ steps.install_kissat_version.outputs.version }}"
+          fi
+          if [ -n "${{ steps.install_cadical_version.outputs.version }}" ]
+          then
+            echo "cadical ${{ steps.install_cadical_version.outputs.version }}"
+          fi
+      - name: Run CBMC proofs
+        id: run_cbmc_proofs
+        shell: bash
+        env:
+          EXTERNAL_SAT_SOLVER: kissat
+        working-directory: ${{ env.PROOFS_DIR }}
+        run: ${{ env.RUN_CBMC_PROOFS_COMMAND }}
+      - name: Set name for zip artifact with CBMC proof results
+        id: artifact
+        if: ${{ env.AWS_CLOUDFRONT_DOMAIN == '' }}
+        run: |
+          echo "name=cbmc_proof_results_${{ fromJson(toJson(github.event.repository)).name }}_$(date +%Y_%m_%d_%H_%M_%S)" >> $GITHUB_OUTPUT
```

Comment:
Because it allows for neat sorting of extracts in a user's downloads folder.

```
cbmc_proof_results_s2n-tls_2022_12_07_12_13_14
cbmc_proof_results_s2n-tls_2022_12_07_21_32_43
```

Reply:
> Right, but there's already an international format for this, we don't need to invent a new one and make people write custom format string to parse it...

What's the right command that we should be using?